tendrl/commons/message.py

```diff
@@ -153,7 +153,7 @@ def serialize_message(obj):
         serial = obj.isoformat()
         return serial
     elif isinstance(obj, Exception):
-        return yaml.safe_dump(obj)
+        return yaml.safe_dump(obj.message)
```

Comment:
No yaml.safe_dump of exception object fails.

>>> ex = Exception('Exception caught')
>>> from ruamel import yaml
>>> yaml.safe_dump(ex)
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "/usr/lib64/python2.7/site-packages/ruamel/yaml/main.py", line 278, in safe_dump
    return dump_all([data], stream, Dumper=SafeDumper, **kwds)
  File "/usr/lib64/python2.7/site-packages/ruamel/yaml/main.py", line 230, in dump_all
    dumper.represent(data)
  File "/usr/lib64/python2.7/site-packages/ruamel/yaml/representer.py", line 45, in represent
    node = self.represent_data(data)
  File "/usr/lib64/python2.7/site-packages/ruamel/yaml/representer.py", line 89, in represent_data
    node = self.yaml_representers[None](self, data)
  File "/usr/lib64/python2.7/site-packages/ruamel/yaml/representer.py", line 325, in represent_undefined
    raise RepresenterError("cannot represent an object: %s" % data)
ruamel.yaml.representer.RepresenterError: cannot represent an object: Exception caught
>>> yaml.safe_dump(ex.message)
'Exception caught\n...\n'